package.json

```diff
@@ -67,7 +67,7 @@
   },
   "dependencies": {
     "@apidevtools/json-schema-ref-parser": "^9.0.6",
-    "@asyncapi/specs": "^2.11.0",
+    "@asyncapi/specs": "asyncapi/spec-json-schemas#2022-01-release",
```

Comment:
@fmvilas are you sure we should grab sources from github?
imho we should either pull a specific version from npm `2.13.0-2022-01-release.1` or just fix to the release candidate tag, so `2022-01-release`

Reply:
Question: If we use `2022-01-release` as version, it will point to the same code as latest commit in that branch? So it is the same as I added but pointing to the npm package tag instead of github branch, right? If so, I can change to it as IMHO it also makes sense as you said.

Reply:
technically if you point to `2022-01-release` and you do `npm install` then `@asyncapi/specs` version `2.13.0-2022-01-release.1` will be installed, if `2` release candidate is available, then `2.13.0-2022-01-release.2` will be installed. 

so yeah, might be that the best solution is to point to the `2022-01-release` npm tag instead of a specific version each time

Reply:
It's done!